src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -76,27 +76,15 @@
     /**
      * Represents the information used to create this shard.
      */
-    class ShardInfo
+    interface ShardInfo
```

Comment:
Why was this changed?

Reply:
because I needed a way to set the total amount of shards while logging in for auto fetching the recommended amount of shards without adding additional REST calls